content/advanced_usage/pipelines_as_code.md

```diff
@@ -70,3 +70,87 @@ Once you've added a config repository, you'll see new pipelines in the pipeline
 As of GoCD 19.1.0, you can export pipeline definitions to a format accepted by the config repository plugins (for instance, the YAML or JSON plugins). You can then check in these pipeline definitions to a source code repository and remove them from GoCD's main XML configuration file.
 
 ![Config repo yaml](../images/advanced_usage/pipeline-export.gif)
+
+
+### Specifying Rules
+
+Starting GoCD 20.2.0, you can defines `rules` on your config repository specifying which environment/pipeline group/pipeline the repository can refer to.
+
+Previously, giving access to a config repository’s backing SCM repository meant giving full access to the GoCD instance, almost. Such users have access to all environments/pipeline group and can, possibly, add tasks to expose the secrets configured.
+
+With `rules`, the owners of the config repositories can define which all environment/pipeline group can the repo add a pipeline to. Similarly, they can also define which pipelines can be referred as an upstream dependency. Hence, providing more control over what all entities can come via the config repository.
```

Comment:
Instead of `owners of the config repositories`, should we say: `administrators who can create/edit config repositories`?

Like:

```
With `rules`, administrators who can create/edit config repositories can limit the environments/pipeline groups the repo can add a pipeline to.
```

Reply:
The reason I wanted to avoid `administrators` is becoz of RBAC changes we introduced in GoCD 19.12.0 for config-repositories. This means there can be non-admin who have `administer` permissions on config-repo.

Reply:
I see them as `administrators who can create/edit config repositories`. That's why I didn't say (generic) admins or super admins, as it would have been before. How about `users who can create/edit config repositories`?

Reply:
I would prefer `users` over `administrators` as we have used `admin` in our docs and that can be a cause of confusion. WDYT?

Reply:
Yes, that makes sense.